webpack/assets/javascripts/react_app/redux/middlewares/DebounceMiddleware/DebounceFixtures.js

```diff
@@ -0,0 +1,12 @@
+import { DEFAULT_DEBOUNCE } from './DebounceConstants';
+import { withDebounce } from './DebounceHelpers';
+
+export const type = 'SOME_TYPE';
+export const key = 'SOME_KEY';
+export const payload = { test: true };
+export const debounce = DEFAULT_DEBOUNCE;
```

Comment:
Maybe here can be a different value so the test could check that the value isn't always the default value?

Reply:
I think that on test there are no meanings to timers as we should use `jest.useFakeTimers`